lblod/frontend-gelinkt-notuleren
# GN-4985: IV LMB queries

### Overview
This PR adds an initial mandatee table configuration containing queries that will be used in the inauguration meeting.
Based on https://github.com/lblod/app-gelinkt-notuleren/pull/191

##### connected issues and PRs:
https://github.com/lblod/app-gelinkt-notuleren/pull/191
[GN-4985](https://binnenland.atlassian.net/browse/GN-4985?atlOrigin=eyJpIjoiYzQxMGRmMTg5NTMzNGY1MDhkZDZjZGI4NTE5NmJmY2UiLCJwIjoiaiJ9)


### Setup
The `vendor-proxy` service should be correctly configured in your stack

### How to test/reproduce
- Start the app
- Create/open an inauguration meeting
- Insert the different type of tables in the different agendapoints
- Notice that they (more-or-less) update correctly when synchronizing the meeting
- Tables containing mandatees should contain the (more-or-less correct) annotations for the mandatees
- Date inside tables should be interactive


### Challenges/uncertainties
- Parts of the code are still marked as `TODO`
- It could be handy to have the ability to add an optional comment and default title to the configurations of the LMB plugin
- I used dutch for the SPARQL variables, as that seemed the most consistent to me.

### Checks PR readiness
